Kconfig.nrf

```diff
@@ -25,6 +25,10 @@ config MBEDTLS_LIBRARY_NRF_SECURITY
 
 endchoice
 
+# nRF Connect SDK default experimental warnings to on.
+config WARN_EXPERIMENTAL
+	default y
+
```

Comment:
I thought it would be default n, and then selected by the features that are experimental?